### PR TITLE
Gracefully handle missing attributes in 'all_input_files'

### PR DESCRIPTION
**Pull request by nsheff**

### CODE REVIEW

**Comment by nsheff** [comment]
Still trying to figure out how to add a test to this but I will need some input from you on how to stick it into the pytest framework.

File: looper/looper.py [hunk 1]
```diff
@@ -630,18 +630,19 @@ def check(prj):
 
 
 def main():
-
     # Parse command-line arguments and establish logger.
     args, remaining_args = parse_arguments()
 
+    
+    _LOGGER.info("Command: " + args.command + "( Looper version: " + __version__ + ")")
```
Comment (vreuter) [inline looper/looper.py]: Can you use `format` and make the whitespace inside the left paren be outside instead?

File: looper/models.py [hunk 2]
```diff
@@ -1303,7 +1303,12 @@ def get_attr_values(self, attrlist):
         values = []
 
         for attr in attribute_list:
-            values.append(getattr(self, attr))
+            if hasattr(self, attr):
+                values.append(getattr(self, attr))
+            else:
+                # If this attribute is not set, return an empty string
```
Comment (vreuter) [inline looper/models.py]: The code makes this clear; only the second comment line is needed.

Comment (nsheff) [inline looper/models.py]: yep!

File: looper/looper.py (hunk 1 above)
Comment (nsheff) [inline looper/looper.py]: I just moved the space...

**Pull request merged**